### DISCUSSION TITLE
Need a way to make a calendar item the full height of the row

### DISCUSSION THREAD

**Discussion by rjray** [namespace-ee/react-calendar-timeline#1003] [Q&A]
**Is your feature request related to a problem? Please describe.**
I have a use-case where I need some of the events that appear in the timeline to be the full height of the row. We use this component in a scheduling application where users click on rows to reserve things, but we want to mark "maintenance windows" where they are unable to make a new reservation.

**Describe the solution you'd like**
A Boolean parameter that could be passed in the call to `getItemProps` that would tell the code to make the item the full height of the row, instead of leaving 5 or so pixels above and below the item.

**Describe alternatives you've considered**
I've tried providing explicit `height` values in the CSS passed to `getItemProps`, but this is overridden by the component code.

**Comment by The-Priya-Saw** [comment]
hello @rjray , can u please assign this to me? I would like to contribute to this project.

**Comment by rjray** [comment]
> hello @rjray , can u please assign this to me? I would like to contribute to this project.

@The-Priya-Saw,

I don't own the project, so I can't assign it. But I'm sure you could fork and start working on it on your own! Then create a pull request that cites this issue number when you're done.

**Comment by VTorres09** [comment]
Any workaround for that?
